manifests/server/export.pp

```diff
@@ -67,7 +67,10 @@
 define nfs::server::export (
   String[1]           $v3_export_name         = $name,
   String[1]           $v4_export_name         = regsubst($name, '.*/(.*)', '\1' ),
-  String[1]           $clients                = 'localhost(ro)',
+  Variant[
+    String[1],
+    Array[String[1]]
```

Comment:
```suggestion
    Array[String[1]],
```

Reply:
Thank you for the suggestion, unfortunately this is no valid syntax.

Would you prefer to write the parameter in one line?

Reply:
Yes, single line is better.

Reply:
I changed the code and updated the `REFERENCE.md`